packages/orbit-components/src/Layout/Layout.stories.tsx

```diff
@@ -81,67 +94,63 @@ export const Mmb = () => {
         </Card>
       </LayoutColumn>
     </Layout>
-  );
-};
-
-Mmb.story = {
-  name: "MMB",
+  ),
 };
 
-export const Customized = () => (
-  <Layout type="Search">
-    <LayoutColumn dataTest="test" as="span">
-      <Card>
-        <CardSection>
-          <CustomDiv />
-        </CardSection>
-      </Card>
-    </LayoutColumn>
-    <LayoutColumn as="span">
-      <Card>
-        <CardSection>
-          <CustomDiv />
-        </CardSection>
-      </Card>
-    </LayoutColumn>
-    <LayoutColumn>
-      <Card>
-        <CardSection>
-          <CustomDiv />
-        </CardSection>
-      </Card>
-    </LayoutColumn>
-  </Layout>
-);
-
-export const Rtl = () => (
-  <RenderInRtl>
+export const Customized: Story = {
```

Comment:
It looks like with the `type="Search"`, the component of LayoutColumn is `aside` which is changed to `span` in the Customized story. It makes sense to my to show that it can be changed but it also feels redundant. Would you rather remove this whole story?